src/assets/icons/filters.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@BoulderBrains You've replaced the exisiting filters.svg file. Need to check that file out from origin/master and put it back in